litestar/serialization/msgspec_hooks.py

```diff
@@ -52,6 +53,8 @@
     deque: list,
     Decimal: lambda val: int(val) if val.as_tuple().exponent >= 0 else float(val),
     Pattern: lambda val: val.pattern,
+    SecretBytes: lambda _: "******",
+    SecretString: lambda _: "******",
```

Comment:
Initially I figured this would be more efficient, but I prob should use the `get_obscured()` method here, as users might subclass and change the obscured representation.